packages/terra-section-header/src/SectionHeader.jsx

```diff
@@ -42,6 +42,11 @@ const propTypes = {
    * Sets the background of the section header to transparent.
    */
   isTransparent: PropTypes.bool,
+  /**
+   * @private
+   * Specifies whether the section header title position should be fixed
```

Comment:
Doesn't this apply to all content, not just the title?

Reply:
Just the title and the accordion icon, if it exists.  I considered that part of the title.

Reply:
That could be included for clarity. Ya never know 😅
Approving either way
```suggestion
   * Specifies whether the positions of the section header title and the caret icon, if present, should be fixed
```